### PR TITLE
Fixes build warning "unmet peer astro@^4.12.0: found 5.0.5"

### PR DESCRIPTION
**Pull request by sebthom**
Support Astro 5.0.x

### CODE REVIEW

File: package/package.json [hunk 1]
```diff
@@ -44,7 +44,7 @@
 	},
 	"type": "module",
 	"peerDependencies": {
-		"astro": "^4.12.0 || ^5.0.0-beta"
```
Comment (florian-lefebvre) [inline package/package.json:47]: @Fryuni I'm surprised this range doesn't work, any idea why?

**Comment by florian-lefebvre** [comment]
@sebthom Thank you for the PR! Would you mind running `pnpm changeset` from the root, select "minor" and write a shrot sentence that states that it adds support for Astro 5?

File: package/package.json (hunk 1 above)
Comment (Fryuni) [reply package/package.json:47]: `^5.0.0-beta` is equivalent to `>=5.0.0-beta <5.0.0`

Going out of beta is considered a breaking change

**Review by florian-lefebvre — APPROVED** [review]
Thanks!

**Pull request merged**